### PR TITLE
Fixed event galary and event description #1989

### PR DESCRIPTION
**Pull request by SinijStitch**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ const FormAddPictures = ({ eventId, updateGallery, picturesCount }: Props) => {
             onFinish={handleSubmit}
         >
 
-            <Form.Item wrapperCol={{ span: 12, offset: 6 }}
+            <Form.Item wrapperCol={{  }}
```
Comment (0l9h) [inline src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx:97]: Why didn't you just delete the wrapperCol property?

File: src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ const FormAddPictures = ({ eventId, updateGallery, picturesCount }: Props) => {
                 </Upload>
             </Form.Item>
 
-            <Form.Item wrapperCol={{ span: 12, offset: 6 }}>
+            <Form.Item wrapperCol={{}}>
```
Comment (0l9h) [inline src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx:118]: Same here

**Review by 0l9h — APPROVED** [review]

File: src/pages/Actions/ActionEvent/EventInfo/EventDetails.less [hunk 3]
```diff
@@ -11,30 +11,35 @@
...
   box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
 }
-
+.descriptions{
+  @media (max-width: 768px)and (min-width: 541px) {
+   
+    margin-left: -82px;
+    margin-right: 87px;
+  }
+
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/EventDetails.less:37]: Do you need this empty line?

File: src/pages/Actions/ActionEvent/EventInfo/EventDetails.less [hunk 4]
```diff
@@ -43,6 +48,11 @@
   font-size: 14px;
   text-align: center;
   font-style: italic;
+  @media  (min-width: 1024px) {
+   
+
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/EventDetails.less:53]: Same. Do you need them?

File: src/pages/Actions/ActionEvent/EventInfo/EventInfo.less [hunk 5]
```diff
@@ -23,9 +23,13 @@
   align-items: center;
   justify-content: center;
 }
-
+.eventActions{
+  @media (max-width: 768px)and (min-width: 541px) {
+    margin-right: 5px;
+  }
+}
 .event-info-wrapper {
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/EventInfo.less:31]: Add empty line here

File: src/pages/Actions/ActionEvent/EventInfo/EventInfo.less [hunk 6]
```diff
@@ -23,9 +23,13 @@
   align-items: center;
   justify-content: center;
 }
-
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/EventInfo.less:26]: Add empty line here

File: src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx [hunk 7]
```diff
@@ -50,14 +50,19 @@ const Gallery = ({eventId, userAccesses}: Props) => {
...
     const removePicture = (pictureId: number) => setPictures(pictures.filter(picture => picture.galleryId !== pictureId))
-    const GalleryAdministration = (): React.ReactNode[] => {
+    const GalleryAdministration  = (): React.ReactNode[] => {
         if (userAccesses["AddPhotos"]) {
             return [
-                <Title level={2} style={{color: '#3c5438'}} key='spinnerTitle'>Адміністрування галереї</Title>,
-                <FormAddPictures eventId={eventId} updateGallery={addPictures} picturesCount={pictures.length}
-                                 key='addPictures'/>,
+                <div className="">
+
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx:57]: Do you need this empty line?

File: src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx [hunk 8]
```diff
@@ -50,14 +50,19 @@ const Gallery = ({eventId, userAccesses}: Props) => {
...
     const addPictures = (uploadedPictures: EventGallery[]) => setPictures(pictures.concat(uploadedPictures));
     const removePicture = (pictureId: number) => setPictures(pictures.filter(picture => picture.galleryId !== pictureId))
-    const GalleryAdministration = (): React.ReactNode[] => {
+    const GalleryAdministration  = (): React.ReactNode[] => {
         if (userAccesses["AddPhotos"]) {
             return [
-                <Title level={2} style={{color: '#3c5438'}} key='spinnerTitle'>Адміністрування галереї</Title>,
-                <FormAddPictures eventId={eventId} updateGallery={addPictures} picturesCount={pictures.length}
-                                 key='addPictures'/>,
+                <div className="">
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx:56]: Why didn`t you delete this className ?

File: src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx [hunk 9]
```diff
@@ -50,14 +50,19 @@ const Gallery = ({eventId, userAccesses}: Props) => {
...
+                    
+                <Title level={3} style={{color: '#3c5438'}} key='spinnerTitle'>Адміністрування галереї</Title>
+                <FormAddPictures eventId={eventId} updateGallery={addPictures} picturesCount={pictures.length} 
+                         key='addPictures'/>
+
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx:61]: empty line

File: src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx [hunk 10]
```diff
@@ -50,14 +50,19 @@ const Gallery = ({eventId, userAccesses}: Props) => {
...
+                <div className="">
+                    
+                <Title level={3} style={{color: '#3c5438'}} key='spinnerTitle'>Адміністрування галереї</Title>
+                <FormAddPictures eventId={eventId} updateGallery={addPictures} picturesCount={pictures.length} 
+                         key='addPictures'/>
+            
                 <PicturesWall pictures={pictures} removePicture={removePicture} key='removePictures'/>
+                </div>
             ];
+
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx:65]: empty line

**Review by dashapenkova — CHANGES_REQUESTED** [review]

File: src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx (hunk 1 above)
Comment (SinijStitch) [reply src/pages/Actions/ActionEvent/EventInfo/FormAddPictures.tsx:97]: its doesn't work without it

File: src/pages/Actions/ActionEvent/EventInfo/EventDetails.less [hunk 11]
```diff
@@ -11,30 +11,34 @@
...
-  align-items: center;
-  justify-content: center;
+  display: block;
+    text-align: center;
 }
 
 .descriptions:hover{
   box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
 }
+.descriptions{
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/EventDetails.less:31]: add empty line

File: src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx [hunk 12]
```diff
@@ -50,14 +50,16 @@ const Gallery = ({eventId, userAccesses}: Props) => {
 
     const addPictures = (uploadedPictures: EventGallery[]) => setPictures(pictures.concat(uploadedPictures));
     const removePicture = (pictureId: number) => setPictures(pictures.filter(picture => picture.galleryId !== pictureId))
-    const GalleryAdministration = (): React.ReactNode[] => {
+    const GalleryAdministration  = (): React.ReactNode[] => {
```
Comment (dashapenkova) [inline src/pages/Actions/ActionEvent/EventInfo/Gallery.tsx:53]: why do you need this extra space?

**Review by dashapenkova — APPROVED** [review]

**Review by Daryna-del — APPROVED** [review]

**Pull request merged**